packages/react-native-bridge/android/build.gradle

```diff
@@ -294,23 +294,35 @@ If they are changed, the isBundleUpToDate flag is switched to false. That flag i
         }
     }
 
-    task cleanupNodeModulesFolder(type: Delete) {
+    task cleanupNodeModulesFolders(type: Delete) {
+        doFirst {
+            println "Deleting node_modules folders"
+        }
         dependsOn bundleUpToDateCheck
         onlyIf { !isBundleUpToDate() }
 
-        delete "$jsRootDir/node_modules"
+        delete "$mobileGutenbergRootDir/node_modules", "$mobileGutenbergRootDir/gutenberg/node_modules"
     }
 
     task resetExtractedRNTools(type: Delete) {
-        println "Deleting temporary folders with extracted RN tools"
+        doFirst {
+            println "Deleting temporary folders with extracted RN tools"
+        }
         delete nodeFolder, npmFolder
     }
 
-    preBuild.dependsOn(cleanupNodeModulesFolder)
-    cleanupNodeModulesFolder.dependsOn(backupHermesDebugAAR)
+    preBuild.dependsOn(cleanupNodeModulesFolders)
+    cleanupNodeModulesFolders.dependsOn(backupHermesDebugAAR)
     backupHermesDebugAAR.dependsOn(backupHermesReleaseAAR)
     backupHermesReleaseAAR.dependsOn(copyJSBundle)
     copyJSBundle.dependsOn(buildJSBundle)
     buildJSBundle.dependsOn(npm_install)
     nodeSetup.dependsOn(resetExtractedRNTools)
+
+    clean {
+        doFirst {
+            println "Executing extended clean task that also deletes node_modules folders"
+        }
+        delete "$mobileGutenbergRootDir/node_modules", "$mobileGutenbergRootDir/gutenberg/node_modules"
```

Comment:
Can we DRY-ify the deletion of the 2 folders to a single function or something? I'm afraid we might modify one code site in the future without adjusting the second. WDYT?

Reply:
Pulled the declaration of the node_modules paths out into a variable like we discussed.

Just noting here for documentation: The reason I'm not re-using a single task is because it gets complicated when you run something like `./gradlew clean installWasabiDebug` because in that situation we're trying to delete the node_modules folders twice (once with the clean and once after the js bundle is built), but if there is only one task it will only be run once.